__tests__/unit/fact.test.js

```diff
@@ -0,0 +1,415 @@
+    let greaterThanMin = getRandomApiNum({ min: 2010, type: "trivia" });
+    expect(greaterThanMin).toBeGreaterThan(2010);
+  });
+
+  test("return random number less than 2010", function () {
+    let lessThanMax = getRandomApiNum({ max: 2010, type: "trivia" });
+    expect(lessThanMax).toBeLessThan(2010);
+  });
+
+  test("return random number when min and max are NaN", function () {
+    let invalidInputNum = getRandomApiNum({ min: "a", max: "b", type: "year" });
+    expect(typeof invalidInputNum).toBe("number");
+  });
+
+  test("return random number when min and max are undefined", function () {
+    let noMinMax = getRandomApiNum({ type: "year" });
+    expect(typeof noMinMax).toBe("number");
+  });
+});
+
+describe("getRandomApiNum() with type 'math'", () => {
+  test("return same number as min and max", function () {
+    let sameAsMinMax = getRandomApiNum({ min: 2010, max: 2010, type: "math" });
+    expect(sameAsMinMax).toEqual(2010);
+  });
+
+  test("return random number greater than 2010", function () {
+    let greaterThanMin = getRandomApiNum({ min: 2010, type: "math" });
+    expect(greaterThanMin).toBeGreaterThan(2010);
+  });
+
+  test("return random number less than 2010", function () {
+    let lessThanMax = getRandomApiNum({ max: 2010, type: "math" });
+    expect(lessThanMax).toBeLessThan(2010);
+  });
+
+  test("return random number when min and max are NaN", function () {
+    let invalidInputNum = getRandomApiNum({ min: "a", max: "b", type: "year" });
+    expect(typeof invalidInputNum).toBe("number");
+  });
+
+  test("return random number when min and max are undefined", function () {
+    let noMinMax = getRandomApiNum({ type: "year" });
+    expect(typeof noMinMax).toBe("number");
+  });
+});
+
+describe("getRandomApiNum() with type 'year'", () => {
+  test("return same number as min and max", function () {
+    let sameAsMinMax = getRandomApiNum({ min: 2010, max: 2010, type: "year" });
+    expect(sameAsMinMax).toEqual(2010);
+  });
+
+  test("return random number greater than 2010", function () {
+    let greaterThanMin = getRandomApiNum({ min: 2010, type: "year" });
+    expect(greaterThanMin).toBeGreaterThan(2010);
+  });
+
+  test("return random number less than 2010", function () {
+    let lessThanMax = getRandomApiNum({ max: 2010, type: "year" });
+    expect(lessThanMax).toBeLessThan(2010);
+  });
+
+  test("return random number when min and max are NaN", function () {
+    let invalidInputNum = getRandomApiNum({ min: "a", max: "b", type: "year" });
+    expect(typeof invalidInputNum).toBe("number");
+  });
+
+  test("return random number when min and max are undefined", function () {
+    let noMinMax = getRandomApiNum({ type: "year" });
+    expect(typeof noMinMax).toBe("number");
+  });
+});
+
+describe("getSentence() for type 'year'", () => {
+  test("When wantFragment is true, returns partial text only", function () {
+    const sentence = getSentence({
+      wantFragment: { fragment: true },
```

Comment:
this looks a bit odd, can you make it so that it just looks like:

```suggestion
      wantFragment: true,
```

Reply:
i hope this will just work, since `getSentence` just checks whether `wantFragment` is undefined or not.